arch/arm/src/cmake/clang.cmake

```diff
@@ -237,11 +237,17 @@ set(PREPROCESS ${CMAKE_C_COMPILER} ${CMAKE_C_FLAG_ARGS} -E -P -x c)
 
 set(NUTTX_FIND_TOOLCHAIN_LIB_DEFINED true)
 
-function(nuttx_find_toolchain_lib)
-  execute_process(
-    COMMAND ${CMAKE_C_COMPILER} ${CMAKE_C_FLAG_ARGS} ${NUTTX_EXTRA_FLAGS}
-            --print-file-name
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-    OUTPUT_VARIABLE extra_lib_path)
-  nuttx_add_extra_library(${extra_lib_path})
-endfunction()
+if(CONFIG_BUILTIN_COMPILER_RT)
+  function(nuttx_find_toolchain_lib)
+
+  endfunction()
+else()
+  function(nuttx_find_toolchain_lib)
```

Comment:
but it affects all extra library

Reply:
Because the last link step of the cmake of clang in master has problems, it cannot be used and can only be linked manually. I just noticed that my last change would cause the rt.builtin_xxx.a file to not be found. I will submit a patch later to modify this place.

Reply:
Ok , please cowork with @xuxin930 fix this problem.